ci_framework/roles/copy_container/files/copy-quay/utils.go

```diff
@@ -189,6 +189,11 @@ func fetchLogs(URL string) string {
         data , _ = _fetch(path)
     }
 
+    if data == "" {
```

Comment:
why is this needed? Care to amend your commit message to explain why you have to modify some go code?

Reply:
Hi Cedric, Quay push job basically checks the containers-built.log file and i am creating that fil in this PR but the path is different so i will need to add that here as well. I will add more info in the commit message once the job will green this PR is still in work in progress phase.